README-CMAKE

```diff
@@ -59,9 +59,12 @@ There are some global CMake options that you can use in Exiv2 :
                          "profile".
                          "relwithdebinfo" : default. use gcc -O2 -g options.
                          "Release"        : generate stripped and optimized bin files. For packaging.
--DBUILD_SHARED_LIBS=1 : Build DLL (=0 for static library)
+-DBUILD_SHARED_LIBS=ON : Build DLL (OFF for static library)
+-DCMAKE_CXX_FLAGS="X"  : In this way you can set specific compiler flags or definitions to the
```

Comment:
I would suggest to reword this as: "... you can **pass** specific compiler flags ..."